pkg/minishift/util/set_proxy_to_shell.go

```diff
@@ -1,32 +0,0 @@
-/*
-Copyright (C) 2016 Red Hat, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package util
```

Comment:
You seem to want to remove this setting of the shell environment, but for example there is still _ shellProxyEnv_.